x-pack/plugins/apm/public/components/shared/span_links/span_links_tab_content.tsx

```diff
@@ -31,6 +31,22 @@ export function getSpanLinksTabContent({
   return {
     id: 'span_links',
     'data-test-subj': 'spanLinksTab',
+    prepend: (
+      <EuiBetaBadge
```

Comment:
Could you use t[his component](https://github.com/elastic/kibana/blob/e9c1c3932b0a8c79c9ad2960fde0fdb1659022f1/x-pack/plugins/apm/public/components/shared/technical_preview_badge.tsx#L12)?

Reply:
yes, done it here, https://github.com/elastic/kibana/pull/132280/commits/5892d250241f7dcca382e15f9b7873a6c784eb11, I didn't know about this component, thanks. I added an optional prop to show an icon instead.

Reply:
Thanks!